deploy.sh

```diff
@@ -10,7 +10,7 @@ export TERRAFORM_ROOT_OPERATIONS="$BITOPS_OPSREPO_ENVIRONMENT_DIR"
 export BITOPS_SCHEMA_ENV_FILE="$TERRAFORM_ROOT_OPERATIONS/ENV_FILE"
 export SCRIPTS_DIR="$TERRAFORM_ROOT_SCRIPTS/scripts"
 
-if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ]; then
+if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ] || [ "$TERRAFORM_SKIP_DEPLOY" == "true" ]; then
   echo "SKIP_DEPLOY_TERRAFORM is set.  Skipping."
```

Comment:
If we're replacing old format ENV `TERRAFORM_SKIP_DEPLOY` with a new one `TERRAFORM_SKIP_DEPLOY ` then message should be updated to a new version too: 
```suggestion
  echo "TERRAFORM_SKIP_DEPLOY is set.  Skipping."
```